src/native/async_callback.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+#include <assert.h>
+#include <jni.h>
+
+#include "async_callback.h"
+
+struct crt_async_callback global_async_callback;
```

Comment:
g_ is sufficient, and matches our coding standard

Reply:
Fixed